.teamcity/pom.xml

```diff
@@ -118,6 +118,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <id>update-test-buckets</id>
+                        <phase>install</phase>
```

Comment:
Why does this have to be in the install phase? Do we need a phase here at all?

Reply:
I thought it need to run at a later phase, but actually it needs `mvn compile exec:java@update-test-buckets`.